src/main/java/twitterapp/src/resources/TwitterAppResource.java

```diff
@@ -53,6 +53,19 @@ public Response getTimeline(){
         }
     }
 
+    @GET
+    @Path("/timeline/user")
+    public Response getUserTimeline(){
+
+        try{
+            Optional<List<TwitterPost>> statuses = service.getUserTimeline();
+
+            return Response.ok(statuses.get(), MediaType.APPLICATION_JSON_TYPE).build();
```

Comment:
What does your code do if statuses is not present? What does optional .get() do in those cases?